### PR TITLE
[testing-devel] manual update of lockfile

### PR DESCRIPTION
**Pull request by dustymabe**
While we currently have the bodhi-updates pipeline down let's manually
update the lockfile so we can get a new build going.

### CODE REVIEW

File: manifest-lock.x86_64.json [hunk 1]
```diff
@@ -642,6 +642,9 @@
     "libsigsegv": {
       "evra": "2.11-8.fc31.x86_64"
     },
+    "libslirp": {
```
Comment (jlebon) [inline manifest-lock.x86_64.json:645]: New package. Worth doing a sanity-check re. size/content, though going to guess that's from podman.

**Review by jlebon — APPROVED** [review]
Thanks for this!

**Comment by dustymabe** [comment]
CI is failing because the new fedora-release build:

```
error: Package 'fedora-release-common' has (currently) unsupported <lua> script in '%posttrans'
```

Looks like a lua scriptlet was added in https://src.fedoraproject.org/rpms/fedora-release/c/bfc4c31d8f625d4963a8169c9ba65686da7a4ce0?branch=f31 and [rpm-ostree doesn't support lua](https://github.com/coreos/rpm-ostree/issues/749). The good news is that it was only implemented for f31.

File: manifest-lock.x86_64.json (hunk 1 above)
Comment (dustymabe) [reply manifest-lock.x86_64.json:645]: about 200K: 

```
$ rpm -ql libslirp | xargs -I{}  bash -c "test -f {} && ls -lh {}"
lrwxrwxrwx. 2 root root 39 Apr 14 14:20 /usr/lib/.build-id/b5/a077ba537e65969c9caeda966f0ca27f228375 -> ../../../../usr/lib64/libslirp.so.0.1.0
lrwxrwxrwx. 2 root root 17 Apr 14 14:20 /usr/lib64/libslirp.so.0 -> libslirp.so.0.1.0
-rwxr-xr-x. 2 root root 194K Jan  1  1970 /usr/lib64/libslirp.so.0.1.0
-rw-r--r--. 2 root root 2.9K Jan  1  1970 /usr/share/licenses/libslirp/COPYRIGHT
```

**Comment by dustymabe** [comment]
> CI is failing because the new fedora-release build:

added a commit to hold the fedora-release rpms back while we sort out https://github.com/coreos/fedora-coreos-tracker/issues/459

**Pull request merged**